packages/block-library/src/container/editor.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block-container .editor-block-list__block {
+	// We need to enforce a cascade by telling nested blocks to inherit
```

Comment:
Where is there a color otherwise being assigned that is breaking the default cascade? It's not clear to me that this should be needed.

Reply:
@aduth See my comment here: https://github.com/WordPress/gutenberg/issues/10067#issuecomment-430287376

We map `body` to `.editor-block-list__block` inside `editor-styles.scss` which means each block resets the cascade [for these properties](https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/editor-styles.scss#L6-L9).

Reply:
That's a bit unfortunate 😕 But OK 👍

Reply:
Agreed we should refactor the editor styles to apply to the editor container. The downside though is that it can affect the UI bits (toolbars, movers...) so we need to be careful there.

Reply:
I almost wonder if there's a way to do a forced reset for Gutenberg chrome? Something to ponder for the future but for now this works :)